static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -1,31 +1,37 @@
+import type { AdSize } from '@guardian/commercial-core';
+import { adSizes } from '@guardian/commercial-core';
+import { getBreakpoint, getViewport } from 'lib/detect-viewport';
 import config from '../../../lib/config';
 import fastdom from '../../../lib/fastdom-promise';
 import { mediator } from '../../../lib/mediator';
 import { spaceFiller } from '../../common/modules/article/space-filler';
-import { adSizes } from '@guardian/commercial-core';
-import { addSlot } from './dfp/add-slot';
-import { trackAdRender } from './dfp/track-ad-render';
-import { createAdSlot } from './dfp/create-slot';
-
 import { commercialFeatures } from '../../common/modules/commercial/commercial-features';
 import { initCarrot } from './carrot-traffic-driver';
-import { getBreakpoint, getTweakpoint, getViewport } from 'lib/detect-viewport';
+import { addSlot } from './dfp/add-slot';
+import { createAdSlot } from './dfp/create-slot';
+import { trackAdRender } from './dfp/track-ad-render';
 
-const isPaidContent = config.get('page.isPaidContent', false);
+const isPaidContent = config.get('page.isPaidContent', false) as boolean;
 
 const adSlotClassSelectorSizes = {
 	minAbove: 500,
 	minBelow: 500,
 };
 
-const insertAdAtPara = (para, name, type, classes, sizes) => {
+const insertAdAtPara = (
+	para: Node,
+	name: string,
+	type: string,
+	classes?: string,
+	sizes?: Record<string, AdSize[]>,
+): void => {
 	const ad = createAdSlot(type, {
 		name,
 		classes,
 		sizes,
 	});
 
-	return fastdom
+	void fastdom
```

Comment:
Should this return as it looks like one of its references is chaining the promise?

```suggestion
	return fastdom
```

Reply:
I think this can be simplified to return void instead.